### PR TITLE
fix(improvements): remove console warnings pt1

### LINKED ISSUE #7959: [StatusApp] Clean up console warnings Pt1
### Description
Clean up as much as ui related console warnings/errors can be cleaned in 8 hours (task to be continued through iterations)

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #7959

### What does the PR do
Status App Clean up console warnings Pt1

### Affected areas
Status App

### CODE REVIEW

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 1]
```diff
@@ -43,18 +43,20 @@ StatusSectionLayout {
...
+        if (!community && !community.tags) {
+            try {
+                const json = JSON.parse(community.tags);
+                const tagsArray = json.map(tag => {
+                                               return tag.name;
+                                           });
+                return JSON.stringify(tagsArray);
+            }
+            catch (e) {
+                console.warn("Error parsing community tags: ", !!community.tags ? community.tags : "", " error: ", e.message)
```
Comment (alexjba) [inline ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:55]: And if we're checking above for valid objects there is no reason to check here also.

**Review by alexjba — COMMENTED** [review]
Nice! One comment.

File: ui/app/AppLayouts/Wallet/stores/RootStore.qml [hunk 2]
```diff
@@ -22,7 +22,7 @@ QtObject {
     property var generatedAccounts: walletSectionAccounts.generated
     property var appSettings: localAppSettings
     property var accountSensitiveSettings: localAccountSensitiveSettings
-    property string locale: appSettings.locale
+    property string locale: !!appSettings.locale ? appSettings.locale : ""
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/stores/RootStore.qml:25]: The real problem here, there's no such thing as `appSettings.locale` :)

It should read:
```suggestion
    property string locale: Qt.locale().name
```

**Review by caybro — CHANGES_REQUESTED** [review]
One minor thing (plus what AlexJb wrote)

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 3]
```diff
@@ -43,18 +43,20 @@ StatusSectionLayout {
     property bool hasAddedContacts: false
 
     readonly property string filteredSelectedTags: {
-        if (!community || !community.tags)
+        if (!community && !community.tags) {
```
[the remark this answers is not in the document]
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml]: done

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml]: done

File: ui/app/AppLayouts/Wallet/stores/RootStore.qml (hunk 2 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Wallet/stores/RootStore.qml]: done

**Review by caybro — APPROVED** [review]

**Review by alexjba — APPROVED** [review]

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml]: @alexjba fyi just pushed one more change to remove
`Warning: Error parsing community tags:  null  error:  Cannot read property 'map' of null (qrc:/app/AppLayouts/Chat/views/CommunitySettingsView.qml:55, expression for filteredSelectedTags)`

Comment (alexjba) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml]: I don't understand how is it entering the if statement with null community.tags if we're checking it for null 🤯

Comment (caybro) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml]: Because `null!==undefined`? :)

Comment (alexjba) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml]: yeah, but both undefined and null should be falsy values.

**Pull request merged**